coverage_point_calculator/src/lib.rs

```diff
@@ -114,8 +114,8 @@ pub struct CoveragePoints {
     pub speedtest_multiplier: Decimal,
     /// Input Radio Type
     pub radio_type: RadioType,
-    /// Input RadioThreshold
-    pub radio_threshold: RadioThreshold,
+    /// Input ServiceProviderBoostedRewardEligibility
+    pub service_provider_boosted_reward_eligibility: ServiceProviderBoostedRewardEligibility,
```

Comment:
This naming convention is very verbose, how about replacing the ServiceProvider chunk with SP ?

Reply:
I actually think the naming is helpful in the coverage point calculator